cmd/kubeadm/app/constants/constants.go

```diff
@@ -479,15 +479,16 @@ var (
 
 	// SupportedEtcdVersion lists officially supported etcd versions with corresponding Kubernetes releases
 	SupportedEtcdVersion = map[uint8]string{
-		22: "3.5.12-0",
-		23: "3.5.12-0",
-		24: "3.5.12-0",
-		25: "3.5.12-0",
-		26: "3.5.12-0",
-		27: "3.5.12-0",
-		28: "3.5.12-0",
-		29: "3.5.12-0",
-		30: "3.5.12-0",
+		22: "3.5.13-0",
+		23: "3.5.13-0",
+		24: "3.5.13-0",
+		25: "3.5.13-0",
+		26: "3.5.13-0",
+		27: "3.5.13-0",
+		28: "3.5.13-0",
+		29: "3.5.13-0",
+		30: "3.5.13-0",
+		31: "3.5.13-0",
```

Comment:
Is this going to be a problem for version 1.30? Whether it is necessary to display the latest version only to 1.30 on the release-1.30 branch?

cc @neolit123

Reply:
yes, should be 1.30 max in the list.
same for older branches.

Reply:
updated.